### PR TITLE
github: Use slint 0.3.0 as basis to run updater_test

### PR DESCRIPTION
**Pull request by hunger**
No need to test the versions in the 0.2 range since the updater has not changed much in that time.

### CODE REVIEW

**Review by ogoffart — APPROVED** [review]

**Review by ogoffart — CHANGES_REQUESTED** [review]
You now what, no need to call the updater anyway since it wouldn't do anything.
the `run the updater` part can just be commented out for when we re-enable it.

Also no need to --skip anything anymore 

The goal being that to test that the old test still work.

**Comment by hunger** [comment]
I also removed the "errors that used to be warnings".

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -316,11 +316,11 @@ jobs:
       CARGO_INCREMENTAL: false
       # There were binding loops with layout in our demo, so ignore theses errors
       # Also ignore errors that were warnings before
-      SLINT_INTERPRETER_ERROR_WHITELIST: "is part of a binding loop;must be called;duplicated element id;Unknown type Clip;in Path"
+      SLINT_INTERPRETER_ERROR_WHITELIST: "is part of a binding loop"
```
Comment (ogoffart) [inline .github/workflows/ci.yaml:319]: Right, none of these error should be there anymore, not even the binding loops, so these lines could go (also the comments)  
But it's ok to keep it there for reference if we need them later.

```suggestion
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -341,23 +341,22 @@ jobs:
...
+    # - name: Show the diff
+    #   run: git diff
     - name: test
       uses: actions-rs/cargo@v1
       with:
           command: test
-          # skip example_printerdemo_old because that was not in 0.0.5
           # skip test_interpreter_examples_rotate because the internal rotate element was removed
-          args: --bin test-driver-interpreter -- --skip example_printerdemo_old --skip test_interpreter_examples_rotate
+          args: --bin test-driver-interpreter -- --skip test_interpreter_examples_rotate
```
Comment (ogoffart) [inline .github/workflows/ci.yaml:362]:
```suggestion
          args: --bin test-driver-interpreter
```

no need to skip anything.  the rotate example was updated to the new API in 0.3  (comment can also go)

**Review by ogoffart — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (hunger) [reply .github/workflows/ci.yaml]: Sorry for pushing another version: GH insists it can not apply your changes:-(

**Review by ogoffart — APPROVED** [review]

**Pull request merged**